build.bat

```diff
@@ -0,0 +1,12 @@
+rem Download submodules.
+if not exist libs\wxWidgets\README.md (
```

Comment:
More semantics: it may be better here to create a file and use that as a marker, instead of relying on README.md in wxWidgets (if they switch to org-mode or RST, we're boned)